examples/threadsafe/src/lib.rs

```diff
@@ -83,6 +86,11 @@ impl ThreadsafeCounter {
             self.count.load(Ordering::SeqCst)
         }
     }
+
+    fn cloneable(self: Arc<Self>) {
```

Comment:
to be clear, I think we want a new example rather than abusing this, but this is OK for demo purposes